### PR TITLE
Update sensitivity module to follow Blue style

### PR DESCRIPTION
**Pull request by ConnectedSystems**
As in title.

### CODE REVIEW

File: src/analysis/sensitivity.jl [hunk 1]
```diff
@@ -1,9 +1,17 @@
 module sensitivity
 
-using Logging
-using DataFrames, NamedDims, AxisKeys
-using Distributions, HypothesisTests, Bootstrap, StaticArrays, FLoops
-
+using
```
Comment (Zapiano) [inline src/analysis/sensitivity.jl:3]: Blue Style Guide requires one using per line in alphabetical order (https://github.com/invenia/BlueStyle#module-imports)

**Comment by Zapiano** [comment]
@ConnectedSystems  since this is updating this whole file: on line 491, the function `_map_outcomes` accepts a parameter `y` which is not being used AND doesn't specify the returning type. Also: why have a function that returns its argument without doing nothing?

File: src/analysis/sensitivity.jl (hunk 1 above)
Comment (ConnectedSystems) [reply src/analysis/sensitivity.jl:3]: I know this, but I think the function import by theme suggestion is a better fit here ("Avoid splitting a single export over multiple lines; either define one export per line, or group them by theme.").

That said, I did a poor job of grouping them by theme.

Comment (Zapiano) [reply src/analysis/sensitivity.jl:3]: 1. Options 1 and 2 below use the same space, so why not the second?

```
# 1
using A
      B
      
# 2
using A
using B
```

2. You can have blocks of `using`'s grouped by theme but inside each block group `using`'s in alphabetical order.

**Comment by ConnectedSystems** [comment]
> @ConnectedSystems since this is updating this whole file: on line 491, the function `_map_outcomes` accepts a parameter `y` which is not being used AND doesn't specify the returning type. Also: why have a function that returns its argument without doing nothing?

It's for flexibility and compatibility.

This line is important:
`all_p_rule = _map_outcomes(y, rule)`

User can pass in a `BitVector` / `Vector{Int64}` instead of a function, thus making `rule` effectively the rule already applied to `y`.

File: src/analysis/sensitivity.jl (hunk 1 above)
Comment (ConnectedSystems) [reply src/analysis/sensitivity.jl:3]: Yes, doing that now! 🤣

**Comment by ConnectedSystems** [comment]
Ready for another look

**Review by Zapiano — APPROVED** [review]

**Pull request merged**